modules/core/src/main/java/org/apache/ignite/internal/visor/verify/VisorIdleVerifyTaskArg.java

```diff
@@ -191,13 +194,15 @@ protected void skipZeros(boolean skipZeros) {
 
     /** */
     protected void cacheFilterEnum(CacheFilterEnum cacheFilterEnum) {
-        this.cacheFilterEnum = (cacheFilterEnum == null ? CacheFilterEnum.DEFAULT : cacheFilterEnum);
+        assert nonNull(cacheFilterEnum);
+
+        this.cacheFilterEnum = cacheFilterEnum;
     }
 
     /**
      * @return Kind fo cache.
      */
-    public CacheFilterEnum cacheFilterEnum() {
+    @NotNull public CacheFilterEnum cacheFilterEnum() {
```

Comment:
We don't use `@NotNull` annotations https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-@Annotations